tests/Functional/Types/StringBindingTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Types;
+
+use Doctrine\DBAL\Driver\SQLSrv\Driver;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+
+class StringBindingTest extends FunctionalTestCase
+{
+    public function testAsciiStringBinding(): void
+    {
+        if (! $this->driverSupportsAsciiBinding()) {
+            self::markTestSkipped('Driver does not support ascii string binding');
+        }
+
+        $statement = $this->connection->prepare('SELECT sql_variant_property(?, \'BaseType\')');
+
+        $statement->bindValue(1, 'test', ParameterType::ASCII_STRING);
+        $results = $statement->execute()->fetchOne();
+
+        self::assertEquals('varchar', $results);
+    }
+
+    private function driverSupportsAsciiBinding(): bool
+    {
+        return $this->connection->getDriver() instanceof Driver
+            || $this->connection->getDriver() instanceof \Doctrine\DBAL\Driver\PDO\SQLSrv\Driver;
```

Comment:
This could also be implemented for MySQL, right?

>  MySQL and Microsoft SQL Server use it to store Unicode data. 

https://wiki.php.net/rfc/extended-string-types-for-pdo

Reply:
Yes possibly - do you have a good way to test this? The method described [here](https://stackoverflow.com/questions/1223211/get-the-type-of-a-variable-in-mysql) doesn't work unfortunately.

Reply:
I don't, sorry. I just meant to clarify why you didn't do it in this PR (which is fine)

Reply:
Ah perfect - I can open a subsequent issue on this.